### ISSUE TITLE
Initializing from HSL and HWB producing different values

### ISSUE THREAD

**Issue by Towanime** [Qix-/color#74]
Hello, ran into this when trying to get a CSS keyword from an object initialized using HSL or HWB notations.

```
var c1 = new Color('brown');
var c2 = new Color('hsl(0, 59%, 41%)');
var c3 = new Color({h: 0, s: 59, l: 41});

```

On the first case, converting it to HSL will give the right result:

```
c1.hslString();
// "hsl(0, 59%, 41%)"
```

Trying to convert the other two won't work:

```
c2.keyword();
// undefined
c3.keyword();
// undefined
```

Comparing the RGB values you can see they are different which makes the keyword method fail:

```
c1.rgb();
// {r: 165, g: 42, b: 42}

c2.rgb();
// {r: 166, g: 43, b: 43}

c3.rgb();
// {r: 166, g: 43, b: 43}
```

**Comment by MoOx** [comment]
Probably a round issue. PR are welcome :)

**Comment by Qix-** [comment]
Going to allow `.raw()` in the new version. That might fix it.

**Comment by Qix-** [comment]
Upon re-reading this, I realize what the issue was. The reason why `.keyword()` wasn't behaving as expected was because `color-convert` required an _exact_ RGB value to make a match or else it returned a null-like value. This was fixed by @joshuacoles in Qix-/color-convert#31.

It'll make its way into the v1.x rewrite. :)
